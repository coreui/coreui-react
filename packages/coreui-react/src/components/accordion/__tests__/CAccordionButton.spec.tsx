import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import { CAccordionButton } from '../index'

test('loads and displays CAccordionButton component', async () => {
  const { container } = render(<CAccordionButton>Test</CAccordionButton>)
  expect(container).toMatchSnapshot()
})

test('CAccordionButton customize', async () => {
  const { container } = render(<CAccordionButton className="bazinga">Test</CAccordionButton>)
  expect(container.firstChild).toHaveClass('bazinga')
  expect(container.firstChild).toHaveClass('accordion-button')
  expect(container).toMatchSnapshot()
})
