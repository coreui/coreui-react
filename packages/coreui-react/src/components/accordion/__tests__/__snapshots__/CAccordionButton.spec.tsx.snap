// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CAccordionButton customize 1`] = `
<div>
  <button
    class="accordion-button collapsed bazinga"
    type="button"
  >
    Test
  </button>
</div>
`;

exports[`loads and displays CAccordionButton component 1`] = `
<div>
  <button
    class="accordion-button collapsed"
    type="button"
  >
    Test
  </button>
</div>
`;
